e2e/module-name-mapper-mock/__tests__/storage/track/Track.test.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const {Track} = require('../../../src/storage/track/Track');
+jest.mock('@@storage/track/Track');
+
+test('relative import', () => {
+  const track = new Track();
+  expect(track.someRandomFunction).not.toBeCalled();
```

Comment:
@grosto, I'm currently experiencing the problem this PR was supposed to solve; my aliased components aren't being mocked. 

Reviewing this expected solution, I suspect it's broken again and was not caught because of this test. I believe this expectation here would always be true event if the mock is not successful. In order for this test to be valuable, _someRandomFunction_ should be called if there is no mock, but I don't see any case where it would be called.

Reply:
@dbroytman could you open up a new issue with a reproducion? Or even better, a PR fixing it 🙂